as/cNFT/assembly/models/market.ts

```diff
@@ -0,0 +1,26 @@
+import { u128 } from 'near-sdk-as'
+
+@nearBindgen
+export class Bid {
+    amount: u128
+    bidder: string
+    recipient: string
+    sell_on_share: u16
+    currency: string
+}
+
+@nearBindgen
+export class Ask {
```

Comment:
No Asks for now in cNFT, planning on deploying it to mainnet. This would just add complexity right now.